### PR TITLE
Remove obsolete code

### PR DESCRIPTION
**Pull request by Tirokk**
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-09T21:30:39Z" title="Monday, September 9th 2019, 11:30:39 pm +02:00">Sep 9, 2019</time>_
_Merged <time datetime="2019-09-10T13:04:47Z" title="Tuesday, September 10th 2019, 3:04:47 pm +02:00">Sep 10, 2019</time>_
---

I believe this is obsolete code. Could someone double-check?

@mattwr18 I tried to see any difference to `master` and I couldn't. Did
I miss sth.?

For context: During my work on https://github.com/Human-Connection/Human-Connection/pull/1517
the webapp was complaining about missing mutations when I changed the
dropdown.

### Issues

- Refactoring

### Todo

- [X] None

### CODE REVIEW

**Comment by Tirokk** [comment]
> [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-10T11:57:19Z" title="Tuesday, September 10th 2019, 1:57:19 pm +02:00">Sep 10, 2019</time>_

---


This code is obsolet. I removed it yesterday already in my PR for the comment list update, except the placeholder. But the placeholder translation is now managed in the editor it self when I think about. So it should be obsolete as well.

**Comment by Tirokk** [comment]
> [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-10T12:01:42Z" title="Tuesday, September 10th 2019, 2:01:42 pm +02:00">Sep 10, 2019</time>_

---


My PR is #1537

PS: I can not edit my comment on iPhone.

File: webapp/components/Editor/Editor.vue [hunk 1]
```diff
@@ -70,7 +69,9 @@ export default {
     }
   },
   computed: {
-    ...mapGetters({ placeholder: 'editor/placeholder' }),
+    placeholder() {
+      return this.$t('editor.placeholder')
+    },
```
Comment (Tirokk) [inline webapp/components/Editor/Editor.vue:74]: > [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-10T13:03:46Z" title="Tuesday, September 10th 2019, 3:03:46 pm +02:00">Sep 10, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -70,7 +69,9 @@ export default {
     }
   },
   computed: {
-    ...mapGetters({ placeholder: 'editor/placeholder' }),
+    placeholder() {
+      return this.$t('editor.placeholder')
+    },
```


much better solution!!
state is the root of all evil!

**Review by Mogge — APPROVED** [review]
Looks good! I can still:
- add comments
- edit comments
- add posts
- edit posts
- see the placeholder whenever the editor is empty
- switch the language and see the correct translation of the placeholder

I'm not sure what exactly `editPending` was supposed to do, though. So I would feel more comfortable merging this if someone else (@mattwr18 ?) could give their approval as well.

**Review by Mogge — APPROVED** [review]
great work @roschaefer!!!
you did make some of the same refactors as @Tirokk, so it feels like you could share the credit :stuck_out_tongue_closed_eyes:

**Pull request closed without merging**